docs/sources/flow/reference/cli/convert.md

```diff
@@ -80,6 +81,8 @@ This includes Prometheus features such as
 and many supported *_sd_configs. Unsupported features in a source config result
 in [errors].
 
+You can find a detailed migration guide for this [here](../../../getting-started/migrating-from-prometheus/)
```

Comment:
Added a link out to the migration guides. If someone is here they may not find them easily otherwise.